vm_setup.pl

```diff
@@ -4,11 +4,15 @@
 
 use strict;
 use warnings;
+
+# ignore warnings when using qw since we are intentionally using commas with qw
+no warnings 'qw';
```

Comment:
This should not be necessary because it appears that qw() is not what should be used in the majority of cases that I'm seeing here.  When you don't want perl do any interpretation of the string use singular quotes like 'thing, thing2' or q(thing, thing2) instead and it won't be necessary to disable these warnings.  When using 'thing, thing2' any single quotes found inside the string would need to be backslash-escaped, so q() is probably better if that occurs often.  Reference 'Quote and Quote-like Operators' in perlop manual for more info.

Reply:
Thanks for this.  I knew there had to be a better way to do this, and q() is exactly what I was looking for.